core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/Purchases.kt

```diff
@@ -252,7 +251,7 @@ public expect class Purchases {
         onSuccess: (storeTransaction: StoreTransaction, customerInfo: CustomerInfo) -> Unit,
         isPersonalizedPrice: Boolean? = null,
         oldProductId: String? = null,
-        replacementMode: GoogleReplacementMode = GoogleReplacementMode.WITHOUT_PRORATION,
+        replacementMode: ReplacementMode? = null,
```

Comment:
Note that I also removed the default value from here. That way, we leave it to the native side to choose the default.